cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsNetworkView.java

```diff
@@ -55,4 +58,8 @@ public String getExistingIGW() {
     public String getExistingVPC() {
         return network.getStringParameter(VPC);
     }
+
+    public String getExistingVpcCidr() {
```

Comment:
NIT: I think the casing of VPC should be consistent. Either VPC or Vpc everywhere. Right now this method and the one above it are inconsistent.

Reply:
Ok, fixed. Thanks.